pkg/signalmeow/groups.go

```diff
@@ -401,14 +498,16 @@ func (cli *Client) fetchGroupByID(ctx context.Context, gid types.GroupIdentifier
 	return group, nil
 }
 
-func (cli *Client) DownloadGroupAvatar(ctx context.Context, group *Group) ([]byte, error) {
```

Comment:
I should probably revert this unnecessary function name change, I had functions with both names at first.